src/test/scala/org/ergoplatform/utils/WalletTestOps.scala

```diff
@@ -29,13 +29,13 @@ trait WalletTestOps extends NodeViewBaseOps {
   def wallet(implicit w: WalletFixture): ErgoWallet = w.wallet
 
   def getTrackedAddresses(implicit w: WalletFixture): Seq[ErgoAddress] =
-    Await.result(w.wallet.trackedAddresses(), awaitDuration)
+    Await.result(w.wallet.trackedAddresses(), defaultAwaitDuration)
```

Comment:
couldn't we replace this by just `await`? 
Same for three more occurences below